### PR TITLE
Run all CircleCI jobs by default

### PR DESCRIPTION
**Pull request by hoechenberger**
### Before merging …

- [ ] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

**Comment by hoechenberger** [comment]
@larsoner please merge if happy

File: .circleci/setup_bash.sh [hunk 1]
```diff
@@ -14,18 +14,40 @@ echo "COMMIT_MESSAGE=$COMMIT_MESSAGE"
...
+#         "$CIRCLE_JOB" != *"ds000247")
+#     ]]; then
+#         echo "Skip detected, exiting job ${CIRCLE_JOB} for PR ${CIRCLE_PULL_REQUEST}."
+#         circleci-agent step halt
+# # Otherwise, run everything
+# elif [[ -v CIRCLE_PULL_REQUEST ]]; then
+#     echo "Running job ${CIRCLE_JOB} for PR ${CIRCLE_PULL_REQUEST}"
+# else
+#     echo "Running job ${CIRCLE_JOB} for ${CIRCLE_BRANCH} branch"
+# fi
```
Comment (larsoner) [inline .circleci/setup_bash.sh:40]: I think we should keep the ability for something like `[circle ds000246]`. I'd make use of it for #581 and #574 for example where one dataset will be the most important and informative to test with

Comment (hoechenberger) [reply .circleci/setup_bash.sh:40]: This ability should still be there, no?? at least that was my intention.

**Comment by hoechenberger** [comment]
@larsoner It appears to be working as intended / as requested by you.

**Comment by larsoner** [comment]
Ahh I misunderstood the deleted lines. Thanks @hoechenberger !

**Pull request merged**